ms2deepscore/train_new_model/data_generators.py

```diff
@@ -77,20 +31,8 @@ def __init__(self, spectrums: List[Spectrum],
         selected_compound_pairs
             SelectedCompoundPairs object which contains selected compounds pairs and the
             respective similarity scores.
-        min_mz
-            Lower bound for m/z values to consider.
-        max_mz
-            Upper bound for m/z values to consider.
-        mz_bin_width
-            Bin width for m/z sampling.
-        intensity_scaling
-            To put more attention on small and medium intensity peaks, peak intensities are
-             scaled by intensity to the power of intensity_scaling.
-        metadata_vectorizer
-            Add the specific MetadataVectorizer object for your data if the model should contain specific
-            metadata entries as input. Default is set to None which means this will be ignored.
-        settings
-            The available settings can be found in GeneratorSettings
+        model_settings
```

Comment:
Maybe rename to ms2deepscore_settings or only settings?
I find it confusing that we take the generator settings out of an object model_settings.

Reply:
Would training_settings work?